keps/sig-release/1733-release-notes/README.md

```diff
@@ -28,13 +28,20 @@ team, this is a visible change large enough to warrant a KEP.
 
 The current release notes process is fraught with inefficiencies that should be streamlined.
 
-- There are two different ways to generate release notes:
-[relnotes](https://github.com/kubernetes/release/blob/master/relnotes) and
-[release-notes](https://github.com/kubernetes/release/tree/master/cmd/release-notes).
-  - Because of this there's duplicate effort maintaining separate tools.
+- There are multiple ways to generate release notes:
+  [relnotes (deprecated)](https://github.com/kubernetes/release/blob/master/relnotes),
```

Comment:
Can we mention in implentation history that this has been removed and drop it from the list?

Reply:
Yes, changed as suggested.